### ISSUE TITLE
master branch is broken at c308094e29a2af0275a766e41785aa12f3e389fa

### ISSUE THREAD

**Issue by syohex** [konoha-project/libzen#15]
master branch was broken at 257647af9e8b7766f13243f59c0524c23a35fab6

```
% make
java version "1.7.0_25"
OpenJDK Runtime Environment (IcedTea 2.3.12) (7u25-2.3.12-4ubuntu3)
OpenJDK 64-Bit Server VM (build 23.7-b01, mixed mode)
Building Java implementation
Buildfile: /home/syohei/Desktop/libzen/build.xml

build:
    [javac] Compiling 147 source files to /home/syohei/Desktop/libzen/build
    [javac] /home/syohei/Desktop/libzen/src/zen/ast/ZenFunctionLiteralNode.java:38: error: class ZenFunctionNode is public, should be declared in a file named ZenFunctionNode.java
    [javac] public class ZenFunctionNode extends ZenNode {
    [javac]        ^
    [javac] /home/syohei/Desktop/libzen/src/zen/ast/ZenFunctionNode.java:38: error: duplicate class: zen.ast.ZenFunctionNode
    [javac] public class ZenFunctionNode extends ZenNode {
    [javac]        ^
    [javac] /home/syohei/Desktop/libzen/src/zen/lang/ZenGrammar.java:314: warning: [deprecation] ReportTokenError1(int,String,String) in ZenTokenContext has been deprecated
    [javac]                 TokenContext.ReportTokenError1(ZenLogger.ErrorLevel, "expected \" to close the string literal", LibZen.SubString(SourceText, start, pos));
    [javac]                             ^
    [javac] /home/syohei/Desktop/libzen/src/zen/lang/ZenGrammar.java:319: warning: [deprecation] ReportTokenError1(int,String,String) in ZenTokenContext has been deprecated
    [javac]         TokenContext.ReportTokenError1(ZenLogger.ErrorLevel, "expected \" to close the string literal", LibZen.SubString(SourceText, start, pos));
    [javac]                     ^
    [javac] 2 errors
    [javac] 2 warnings

BUILD FAILED
/home/syohei/Desktop/libzen/build.xml:13: Compile failed; see the compiler error output for details.

Total time: 1 second
make: *** ["./libzen.jar"] Error 1
```

**Comment by syohex** [comment]
This is fixed at c7e4a6a869716bd1442435180d082a68f5a6c643

**Comment by imasahiro** [comment]
This issue is fixed at c7e4a6a8
